utility/src/main/java/org/oppia/android/util/parser/html/BulletTagHandler.kt

```diff
@@ -30,4 +31,13 @@ class BulletTagHandler : CustomHtmlContentHandler.CustomTagHandler {
       }
     }
   }
+
+  /**
+   * Appends a new line to [output] if it doesn't already end in a new line
```

Comment:
Please provide context & links as to why this is correct, and why it must be used.

Also, can you add tests for this to ensure that the newline is added/not added in the corresponding cases? It's important that we ensure this works correctly.

Reply:
Done with adding context and tests.